package.json

```diff
@@ -130,9 +131,11 @@
     "superagent": "^3.8.3",
     "supertest": "^3.1.0",
     "ts-jest": "^24.0.2",
+    "ts-node": "^8.4.1",
     "ts-transformer-imports": "^0.4.3",
     "ttypescript": "^1.5.7",
-    "typescript": "^3.5.3"
+    "typescript": "^3.5.3",
+    "webpack": "^4.40.2"
```

Comment:
Why is webpack being added?

Reply:
There were peer dependency warnings.

Reply:
But isn't that something that the user of the library needs to deal with? We only add peer deps here to get the tests working. The actual users of the library install firebase themselves.

Reply:
They have nothing to do with firebase. There were other libraries probably babel loader that had a peer dep on it, so I added to dev deps. Would you like me to remove them?

Reply:
I think let's keep the deps to a minimum here.  We're only upgrading firebase because the tests don't run on Node v12 correct?

Reply:
Upgrading firebase because `npm i` fails.

Reply:
Got it.  In that case let's just upgrade firebase without adding any additional deps that are not needed here.

Reply:
Removed.